docs/apidocs/qiskit.chemistry.components.bosonic_basis.rst

```diff
@@ -0,0 +1,6 @@
+.. _qiskit-chemistry-components-bosonic_basis:
```

Comment:
Currently `make html` fails because this is not included in any table of contents. I don't know how to fix this so could someone please address this?

Reply:
Yes I can fix the docs

Reply:
I think I found it. We'll see in the next CI run I guess :D